### PR TITLE
GH-7521: Enabled `Select All` in the `Output`

### LINKED ISSUE #7521: As a user I want to copy the content of the current Output channel with `Ctrl/Cmd+A`
### Description
 

When the Output channel is opened and is the active widget, I want to be able to copy the content (`Select All`) with the <kbd>Ctrl/Cmd</kbd>+<kbd>A</kbd> keybindings.

Inspired by: https://github.com/arduino/arduino-pro-ide/issues/37

### Reproduction Steps

**OS and Theia version:**

**Diagnostics:**

### PR DESCRIPTION
**Pull request by kittaakos**
#### What it does

It enables the `Select All` functionality in the `Output` widget. Enabled only if the `Output` widget is the active widget in the shell. It should work with <kbd>Ctrl/Cmd</kbd>+<kbd>A</kbd> keybindings by default. Closes #7521.

#### How to test

 - Make you have some input for the `Output` widget. You can use the `API Sample: Log to Output` command.
 - Activate the widget and try to select all with the keybinding.
 - Make sure the `Output` widget is visible but not active
    - perform a `Select All` via the keyboard whilst the editor is active,
    - perform a `Select All` via the keyboard whilst the SIW widget is active and `input` has the focus.
 - Verify the same in electron.
 
#### Review checklist

- [x] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

**Comment by kittaakos** [comment]
I am going to drop https://github.com/eclipse-theia/theia/pull/7523/commits/7d97ebe59962bcda3cdb87cf8ff75a1798b30bdb once the review is over.

File: packages/output/src/browser/output-widget.tsx [hunk 1]
```diff
@@ -78,12 +78,31 @@ export class OutputWidget extends ReactWidget {
...
         if (this.outputChannelManager.selectedChannel) {
             this.outputChannelManager.selectedChannel.clear();
         }
     }
 
+    selectAll(): void {
+        if (this.outputChannelManager.selectedChannel) {
+            const element = document.getElementById(OutputWidget.IDs.CONTENTS);
+            if (element) {
+                const selectElementContent = (what: HTMLElement) => {
```
Comment (vince-fugnitto) [inline packages/output/src/browser/output-widget.tsx:91]: [minor] perhaps we can find a better name instead of `what`.

Comment (kittaakos) [reply packages/output/src/browser/output-widget.tsx:91]: What do you suggest?

Comment (vince-fugnitto) [reply packages/output/src/browser/output-widget.tsx:91]: > What do you suggest?

I'm not sure, either `content` or even `htmlElement`?

**Comment by vince-fugnitto** [comment]
> Question:
> Could it be a side effect if I "Select All" from the Problem view. It looks like it selects all the "Problem" view content as well as selecting all line ranges in the open editor

@lmcbout it is not a side-effect of this pull-request since the bug currently occurs on `master`. 
We will need to address the 'select all' for the problems-view as well similarly to this pull-request.

**Comment by kittaakos** [comment]
Thank you for the review, @vince-fugnitto, and @lmcbout 👍

**Comment by lmcbout** [comment]
>@lmcbout it is not a side-effect of this pull-request since the bug currently occurs on master.
We will need to address the 'select all' for the problems-view as well similarly to this pull-request.

@vince-fugnitto is there an issue where the "Select all" for the Problem" view is tracked

**Comment by kittaakos** [comment]
> @vince-fugnitto is there an issue where the "Select all" for the Problem" view is tracked

No, but there is a conversation here: https://github.com/eclipse-theia/theia/issues/7525#issuecomment-610861694

**Pull request merged**